shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/distsql/exception/rule/DuplicateKeyGeneratorException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.infra.distsql.exception.rule;
+
+import java.util.Collection;
+
+/**
+ * Duplicate key generator exception.
+ */
+public final class DuplicateKeyGeneratorException extends RuleDefinitionViolationException {
+    
+    private static final long serialVersionUID = 4965160371403179153L;
+    
+    public DuplicateKeyGeneratorException(final String ruleType, final String schemaName, final Collection<String> keyGeneratorNames) {
+        super(1113, String.format("Duplicate %s key generator names `%s` in schema `%s`", ruleType, keyGeneratorNames, schemaName));
```

Comment:
The errorCode 1113 exists in `DuplicateRuleException`, please change to another one.

Reply:
> The errorCode 1113 exists in `DuplicateRuleException`, please change to another one.

Done.